app/scripts/components/analysis/results/chart-card.tsx

```diff
@@ -101,7 +101,7 @@ const getNoDownloadReason = ({ status, data }: TimeseriesData) => {
   if (status === 'loading') {
     return 'Download will be available once the data finishes loading.';
   }
-  if (!data.timeseries.length) {
+  if (!data.timeseries?.length) {
```

Comment:
📓 : Optionally chaining for type safety and ts complaints but not rendering the datasets in the `define` step protects against this already. Users should not be able to render results of datasets without summaries through ui flow